x/cronos/keeper/precompiles/relayer.go

```diff
@@ -155,6 +159,8 @@ func (bc *RelayerContract) Run(evm *vm.EVM, contract *vm.Contract, readonly bool
 		res, err = exec(bc.cdc, stateDB, contract.CallerAddress, precompileAddr, input, bc.ibcKeeper.Timeout, converter)
 	case prefixTimeoutOnClose:
 		res, err = exec(bc.cdc, stateDB, contract.CallerAddress, precompileAddr, input, bc.ibcKeeper.TimeoutOnClose, converter)
+	case prefixTransfer:
+		res, err = exec(bc.cdc, stateDB, contract.CallerAddress, precompileAddr, input, bc.ibctransferKeeper.Transfer, converter)
```

Comment:
Isn't it part of RecvPacket?

Reply:
it's test from another cronos using [transfer](https://github.com/crypto-org-chain/cronos/pull/1219/files#diff-4e1b37cfd1eafd2f519334d816882b899e507a40f73f30124b9ca0a659e7fccfR31)

Reply:
transfer is an end user transaction?

Reply:
I see, need handle by user